### PR TITLE
Here, now stop being on fire..

### PR DESCRIPTION
**Pull request by Aurorablade**
:cl: Fethas
tweak: PLASMAMAN ATMOS SUITS ARE NOW THE SAME AS REGULAR ATMOS SUITS FLAG WISE! AND SO IS THE CE SUIT
/ :cl:

### CODE REVIEW

**Comment by granodd** [comment]
So does this mean that plasma men suits are now fireproof or does it mean that plasma men wearing atmos suits or the CE suit won't be on fire anymore?

**Comment by Aurorablade** [comment]
it means they won't need to wear another suit. the stats are the same.

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -107,14 +107,16 @@
 	name = "plasmaman atmospheric suit"
 	icon_state = "plasmamanAtmos_suit"
 	armor = list(melee = 10, bullet = 5, laser = 10, energy = 5, bomb = 10, bio = 100, rad = 0)
-	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	heat_protection = HEAD												//Uncomment to enable firesuit protection
```
Comment (Fox-McCloud) [inline code/modules/clothing/spacesuits/plasmamen.dm]: I'm pretty sure parent has this.

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 2]
```diff
@@ -107,14 +107,16 @@
...
+	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 
 /obj/item/clothing/head/helmet/space/eva/plasmaman/atmostech
 	name = "plasmaman atmospheric helmet"
 	icon_state = "plasmamanAtmos_helmet0"
 	base_state = "plasmamanAtmos_helmet"
 	armor = list(melee = 10, bullet = 5, laser = 10, energy = 5, bomb = 10, bio = 100, rad = 0)
-	max_heat_protection_temperature = FIRE_HELM_MAX_TEMP_PROTECT
+	heat_protection = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS					//Uncomment to enable firesuit protection
```
Comment (Fox-McCloud) [inline code/modules/clothing/spacesuits/plasmamen.dm]: Same here.

Comment (Aurorablade) [inline code/modules/clothing/spacesuits/plasmamen.dm]: @Fox-McCloud i am not sure cuase something is making palsmatechs change suits. let me look.

Comment (Aurorablade) [inline code/modules/clothing/spacesuits/plasmamen.dm]: Okay helmet head DOES NOT the suit DOES

**Comment by Anticept** [comment]
Plasmemes rejoice?

**Comment by Anoonki** [comment]
I guarantee plasma atmos techs and CEs are gonna still probably wear the regular counterparts just for the sake of being "le funny burning guy xDDD"

**Comment by Aurorablade** [comment]
and then they can be arrested for cuasing a workplace hazard!

**Comment by Anoonki** [comment]
But the fire's only a visual effect, and doesn't actually ignite anything, or act like fire. 
:thinking:

**Comment by Aurorablade** [comment]
that plasma fire lastnight might disagree with you....

**Comment by Fox-McCloud** [comment]
👍

**Comment by Anoonki** [comment]
Huh, maybe it changed, but I recall standing in a room full of plasma as a burning plasman CE and not actually igniting the fire.

**Pull request merged**